### PR TITLE
Docs 2979: Review payment method URLs

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/payments/methods/credit-and-debit-cards/american-express/overview.md [hunk 1]
```diff
@@ -5,11 +5,12 @@ weight: 10
 meta_title: "American Express overview - MultiSafepay Docs"
 short_description: "Key information, supported countries, currencies, and features"
 layout: 'child'
-url: '/payments/methods/amex/overview/'
+url: '/payment-methods/amex/overview/'
```
Comment (Sarah-MultiSafepay) [inline content/payments/methods/credit-and-debit-cards/american-express/overview.md:8]: The main URL is still: /payment-methods/american-express/
Should we change it to amex?
And the activation page is: /payment-methods/american-express/activation/ which doesn't match the rest.

**Review by Sarah-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/payments/methods/credit-and-debit-cards/american-express/overview.md (hunk 1 above)
Comment (FrancescaNCook) [reply content/payments/methods/credit-and-debit-cards/american-express/overview.md:8]: Good spotting! Fixed.

File: content/payments/methods/credit-and-debit-cards/mastercard/_index.md [hunk 2]
```diff
@@ -8,10 +8,12 @@ intro_description: "Mastercard is a leading global credit card scheme. An additi
 layout: 'single'
 logo: '/logo/Payment_methods/Mastercard.svg' 
 short_description: 'Leading global credit card scheme.'
+url: '/payment-methods/mastercard/'
 aliases:
     - /support-tab/magento2/payment-methods/mastercard
     - /payment-methods/mastercard/
     - /payment-methods/credit-and-debit-cards/mastercard
+    - /payments/methods/credit-and-debit-cards/mastercard/
```
[the remark this answers is not in the document]
Comment (FrancescaNCook) [reply content/payments/methods/credit-and-debit-cards/mastercard/_index.md:16]: @Sarah-MultiSafepay  Not quite sure I understand your comment, sorry? The alias in your-logo-online-banking.md is part of a much longer former URL: /payments/methods/credit-and-debit-cards/mastercard/user-guide/displaying-your-logo-in-online-banking/ ?

[the remark this answers is not in the document]
Comment (Sarah-MultiSafepay) [reply content/payments/methods/credit-and-debit-cards/mastercard/_index.md:16]: @FrancescaNCook, not sure anymore! Looks fine now though.

File: content/api/reference/customer-object.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ Default: `en_US` (American English).
 `ip_address` | string | required / recommended 
 
 The customer's IP address.  
-Required for [pay later methods](/payments/methods/pay-later/) and [credit cards](/payments/methods/credit-and-debit-cards/) as part of our [fraud check](/credit-cards-user-guide/uncleared-transactions/), optional but recommended for other payment methods.  
+Required for [pay later methods](/payments/methods/pay-later/) and [credit cards](/payment-methods/credit-debit-cards/) as part of our [fraud check](/glossaries/credit-cards/#uncleared-transactions), optional but recommended for other payment methods.
```
Comment (Sarah-MultiSafepay) [inline content/api/reference/customer-object.md:59]:
```suggestion
Required for [pay later methods](/payment-methods/pay-later/) and [credit cards](/payment-methods/credit-debit-cards/) as part of our [fraud check](/glossaries/credit-cards/#uncleared-transactions), optional but recommended for other payment methods.  
```

File: content/getting-started/glossary/multisafepay-glossary.md [hunk 4]
```diff
@@ -36,13 +36,13 @@ A merchant's bespoke integration with MultiSafepay's services built using [our A
...
 ### ecommerce platform
-An online [platform](/payments/integrations/ecommerce-platforms/) merchants use for accepting payments, e.g. website, mobile app, [PWA](/payments/integrations/pwa/). 
+An online [platform](/integrations/ready-made/) merchants use for accepting payments, e.g. website, mobile app, PWAs. 
 
 ### fraud
 When a person tries to receive products without paying for them by tricking a customer or merchant.
 
 ### integration
-Software merchants use to sell products and services and accept payments, e.g website, mobile app, [PWA](/payments/integrations/pwa/). Merchants may use [ecommerce platforms](/payments/integrations/ecommerce-platforms/) or build a custom integration.
+Software merchants use to sell products and services and accept payments, e.g website, mobile app, [PWA](/payments/integrations/pwa/). Merchants may use [ecommerce platforms](/integrations/ready-made/) or build a custom integration.
```
Comment (Sarah-MultiSafepay) [inline content/getting-started/glossary/multisafepay-glossary.md:45]:
```suggestion
Software merchants use to sell products and services and accept payments, e.g website, mobile app, [PWA](/payments/integrations/pwa/). Merchants may use [ecommerce platforms](/integrations/ready-made/) or [build their own integration](/integrations/self-made/).
```

File: content/my-account/manage-account/managing-your-balance.md [hunk 5]
```diff
@@ -43,6 +43,6 @@ There you can see your:
 
 ## Viewing your uncleared balance
 
-You can set your daily balance report to include the balance of uncleared transactions. This is particularly useful if you offer [pay later](/payments/methods/pay-later/) and/or [credit card](/payments/methods/credit-and-debit-cards/) payment methods. 
+You can set your daily balance report to include the balance of uncleared transactions. This is particularly useful if you offer [pay later](/payments/methods/pay-later/) and/or [credit card](/payment-methods/credit-debit-cards/) payment methods.
```
Comment (Sarah-MultiSafepay) [inline content/my-account/manage-account/managing-your-balance.md:46]:
```suggestion
You can set your daily balance report to include the balance of uncleared transactions. This is particularly useful if you offer [pay later](/payment-methods/pay-later/) and/or [credit card](/payment-methods/credit-debit-cards/) payment methods. 
```

File: content/payments/features/recurring-payments/_index.md [hunk 6]
```diff
@@ -180,12 +180,12 @@ You must have **both** credit card payments and recurring payments enabled for y
...
-- [PrestaShop 1.6](/payments/integrations/ecommerce-platforms/prestashop-1-6)
-- [PrestaShop 1.7](/payments/integrations/ecommerce-platforms/prestashop-1-7)
-- [Shopware 6](/payments/integrations/ecommerce-platforms/shopware6) 
-- [Woocommerce](/payments/integrations/ecommerce-platforms/woocommerce/) 
+- [Magento 1](/magento1)
+- [Magento 2](/magento2)
+- [PrestaShop 1.6](/prestashop-1-6)
+- [PrestaShop 1.7](/prestashop-1-7)
+- [Shopware 6](/shopware-6) 
+- [Woocommerce](/woocommerce/)
```
Comment (Sarah-MultiSafepay) [inline content/payments/features/recurring-payments/_index.md:188]:
```suggestion
- [Magento 1](/magento-1)
- [Magento 2](/magento-2)
- [PrestaShop 1.6](/prestashop-1-6)
- [PrestaShop 1.7](/prestashop)
- [Shopware 6](/shopware-6) 
- [Woocommerce](/woo-commerce/) 
```

**Review by Sarah-MultiSafepay — CHANGES_REQUESTED** [review]
A few suggested changes!
Could you also: 
- Fix `/developer/api/difference-between-direct-and-redirect` in `payment_methods.md`
- Change ZilverCMS contact URL in `zilvercms/index.md` to `https://www.zilverhq.nl/contact`

**Pull request merged**